gnd/src/main/java/com/google/android/gnd/repository/OfflineAreaRepository.java

```diff
@@ -164,25 +164,26 @@ public Completable addAreaAndEnqueue(LatLngBounds bounds) {
     return localDataStore.getOfflineAreaById(offlineAreaId);
   }
 
-  public Flowable<ImmutableSet<Tile>> getIntersectingDownloadedTilesOnceAndStream(
+  public Flowable<ImmutableSet<TileSource>> getIntersectingDownloadedTileSourcesOnceAndStream(
       OfflineArea offlineArea) {
-    return getBaseMapTiles(offlineArea)
+    return getBaseMapTileSources(offlineArea)
         .flatMapPublisher(
             tiles ->
-                getDownloadedTilesOnceAndStream()
+                getDownloadedTileSourcesOnceAndStream()
                     .map(ts -> stream(ts).filter(tiles::contains).collect(toImmutableSet())))
-        // If no tiles are found, we report the area takes up 0.0mb on the device.
-        .doOnError(throwable -> Timber.d(throwable, "no tiles found for area %s", offlineArea))
+        // If no tile sources are found, we report the area takes up 0.0mb on the device.
+        .doOnError(
+            throwable -> Timber.d(throwable, "no tile sources found for area %s", offlineArea))
         .onErrorReturn(__ -> ImmutableSet.of());
   }
 
-  public Flowable<ImmutableSet<Tile>> getDownloadedTilesOnceAndStream() {
+  public Flowable<ImmutableSet<TileSource>> getDownloadedTileSourcesOnceAndStream() {
```

Comment:
This case is a little be trickier - it's not clear how to describe that we're "getting all tile sources for which all tiles have been downloaded", so I think it's ok as-is.